castai/resource_workload_scaling_policy.go

```diff
@@ -449,10 +513,28 @@ func toWorkloadScalingPolicies(obj map[string]interface{}) sdk.Workloadoptimizat
 	if v, ok := obj["max"].(float64); ok && v > 0 {
 		out.Max = lo.ToPtr(v)
 	}
+	if v, ok := obj["limit"].([]any); ok && len(v) > 0 {
```

Comment:
`FieldLimitStrategy` vs `"limit"`

Reply:
I suppose here prior map access was with a const but now we have a nicely named one

Reply:
fixed 👍